cmd/loom/loom.go

```diff
@@ -1133,9 +1134,12 @@ func deployContract(
 		return err
 	}
 
-	err = registry.Register(contractCfg.Name, addr, addr)
-	if err != nil {
-		return err
+	conf, err := common.ParseConfig()
```

Comment:
We should not reparse the config file here. The config should be passed as a function argument.